*** fo_FO.orig.aff	Wed Aug 31 22:02:11 2005
--- fo_FO.aff	Wed Aug 31 22:02:11 2005
***************
*** 6 ****
--- 6,14 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗ÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD	'-
