*** de_DE.orig.aff	Fri Jun 17 10:01:18 2005
--- de_DE.aff	Mon Jun 27 19:42:20 2005
***************
*** 3,4 ****
--- 3,10 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗ÿ
+ 
+ MIDWORD	'
+ 
  # (c) copyright by Bjoern Jacke <bjoern@j3e.de>
*** de_DE.orig.dic	Fri Jun 17 10:01:27 2005
--- de_DE.dic	Sat Jun 18 19:57:07 2005
***************
*** 76258,76259 ****
  zynismusfצrdernd/A
! zzgl
\ No newline at end of file
--- 76258,76260 ----
  zynismusfצrdernd/A
! zzgl
! 
