*** en_CA.orig.aff	Fri Apr 15 13:20:36 2005
--- en_CA.aff	Sat Jun 18 19:42:19 2005
***************
*** 3,4 ****
--- 3,8 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗
+ 
  PFX A Y 1
***************
*** 30,33 ****
  SFX N   e     ion        e
! SFX N   y     ication    y 
! SFX N   0     en         [^ey] 
  
--- 34,37 ----
  SFX N   e     ion        e
! SFX N   y     ication    y
! SFX N   0     en         [^ey]
  
***************
*** 40,42 ****
  SFX H   y     ieth       y
! SFX H   0     th         [^y] 
  
--- 44,46 ----
  SFX H   y     ieth       y
! SFX H   0     th         [^y]
  
***************
*** 47,49 ****
  SFX G   e     ing        e
! SFX G   0     ing        [^e] 
  
--- 51,53 ----
  SFX G   e     ing        e
! SFX G   0     ing        [^e]
  
*** en_CA.orig.dic	Sat Apr 16 14:40:06 2005
--- en_CA.dic	Tue Jun 21 20:25:01 2005
***************
*** 46,48 ****
  R/G
- S
  easternmost
--- 46,47 ----
***************
*** 59,61 ****
  a
! b/KGDT
  Emmey/M
--- 58,61 ----
  a
! probing
! probed
  Emmey/M
***************
*** 82,84 ****
  m/XG
! n/FKT
  o
--- 82,84 ----
  m/XG
! pron
  o
***************
*** 86,91 ****
  fitting/PSY
- p/KRT
  q
! r/GVTJ
! s/FK
  fatting
--- 86,89 ----
  fitting/PSY
  q
! cons
  fatting
***************
*** 94,96 ****
  oedipal
- u
  v/VTK
--- 92,93 ----
***************
*** 3145,3146 ****
--- 3142,3144 ----
  semester/MS
+ etc.
  etch/GZSRDJ
***************
*** 7022,7024 ****
  DA
- DB
  DC
--- 7020,7021 ----
***************
*** 8388,8390 ****
  slotting
- ON
  OR
--- 8385,8386 ----
***************
*** 9125,9127 ****
  perchance
- rte
  hastiness/MS
--- 9121,9122 ----
***************
*** 10603,10604 ****
--- 10598,10602 ----
  dB/M
+ dBi
+ dBm
+ dBd
  Hewet/M
***************
*** 10615,10617 ****
  Garold/M
- db
  tollhouse/M
--- 10613,10614 ----
***************
*** 11017,11019 ****
  hr
- ht
  MCI/M
--- 11014,11015 ----
***************
*** 11609,11611 ****
  demureness/SM
! nd/A
  MIA
--- 11605,11607 ----
  demureness/SM
! nd
  MIA
***************
*** 13669,13671 ****
  engross/LDRSG
! hobbit
  certainty/MUS
--- 13665,13667 ----
  engross/LDRSG
! hobbit/MS
  certainty/MUS
***************
*** 14434,14435 ****
--- 14430,14432 ----
  pompom/MS
+ pompon/M
  Devland/M
***************
*** 19265,19267 ****
  bloodstone/M
! cetera/S
  storm/SGZRDM
--- 19262,19264 ----
  bloodstone/M
! et cetera/S
  storm/SGZRDM
***************
*** 20162,20164 ****
  Hansel/M
! ring/GZJDRM
  Hansen/M
--- 20159,20161 ----
  Hansel/M
! ring/GZJDRMS
  Hansen/M
***************
*** 28157,28158 ****
--- 28154,28156 ----
  pneumonia/MS
+ pneumonic
  Socratic/S
***************
*** 34999,35001 ****
  claque/MS
- etc
  Chad/M
--- 34997,34998 ----
***************
*** 36707,36708 ****
--- 36704,36706 ----
  Moody/M
+ Moolenaar/M
  Bresenham/M
***************
*** 50272,50273 ****
--- 50270,50272 ----
  Dutch/M
+ Farsi
  Sharon/M
***************
*** 52565,52567 ****
  hatchery/MS
! vim/SM
  compatriot/MS
--- 52564,52566 ----
  hatchery/MS
! Vim/SM
  compatriot/MS
***************
*** 53490,53491 ****
--- 53489,53491 ----
  unsearchable
+ searchable
  felicitous/IY
***************
*** 62341 ****
--- 62341,62342 ----
  data/M
+ et al.
