*** en_US.orig.aff	Fri Apr 15 13:20:36 2005
--- en_US.aff	Sat Apr 23 19:57:52 2005
***************
*** 3,4 ****
--- 3,8 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ 
  PFX A Y 1
***************
*** 30,33 ****
  SFX N   e     ion        e
! SFX N   y     ication    y 
! SFX N   0     en         [^ey] 
  
--- 34,37 ----
  SFX N   e     ion        e
! SFX N   y     ication    y
! SFX N   0     en         [^ey]
  
***************
*** 40,42 ****
  SFX H   y     ieth       y
! SFX H   0     th         [^y] 
  
--- 44,46 ----
  SFX H   y     ieth       y
! SFX H   0     th         [^y]
  
***************
*** 47,49 ****
  SFX G   e     ing        e
! SFX G   0     ing        [^e] 
  
--- 51,53 ----
  SFX G   e     ing        e
! SFX G   0     ing        [^e]
  
*** en_US.orig.dic	Fri Apr 15 13:20:36 2005
--- en_US.dic	Tue Apr 19 23:08:36 2005
***************
*** 16911,16912 ****
--- 16911,16913 ----
  dusty/RPT
+ Farsi
  Dutch/M
***************
*** 25963,25965 ****
  hobbing
! hobbit
  hobbler/M
--- 25964,25966 ----
  hobbing
! hobbit/MS
  hobbler/M
***************
*** 36605,36606 ****
--- 36606,36608 ----
  Moog
+ Moolenaar/M
  moo/GSD
***************
*** 38871,38873 ****
  NSF
! n/T
  NT
--- 38873,38875 ----
  NSF
! n
  NT
***************
*** 46507,46509 ****
  Renault/MS
- rend
  renderer/M
--- 46509,46510 ----
***************
*** 48085,48087 ****
  Ryun/M
- S
  SA
--- 48086,48087 ----
***************
*** 54450,54452 ****
  swung
! s/XJBG
  sybarite/MS
--- 54450,54452 ----
  swung
! sings
  sybarite/MS
***************
*** 59538,59540 ****
  vi/MDR
! vim/MS
  vinaigrette/MS
--- 59538,59540 ----
  vi/MDR
! Vim/MS
  vinaigrette/MS
