*** fr_FR.orig.aff	Sun Jul  3 19:34:20 2005
--- fr_FR.aff	Sun Jul 31 22:17:53 2005
***************
*** 3,4 ****
--- 3,24 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗ÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD	'
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
+ 
  PFX A Y 10
***************
*** 690,694 ****
  SFX q   ssait      raient     ssait
- 
- 
- 
- 
--- 710 ----
