*** sv_SE.orig.aff	2010-05-20 21:31:54.000000000 +0200
--- sv_SE.aff	2010-05-20 21:38:36.000000000 +0200
***************
*** 6,7 ****
--- 6,21 ----
  NOSUGGEST !
+ FOL  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ‗ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
  
***************
*** 649 ****
--- 663,664 ----
  SFX  t  el  la/WXZ  el
+ 
*** sv_SE.orig.dic	2010-05-20 21:31:54.000000000 +0200
--- sv_SE.dic	2010-05-20 21:31:54.000000000 +0200
***************
*** 11786,11788 ****
  DVD
! dvs
  dvהljas
--- 11786,11788 ----
  DVD
! #dvs Removed by Stefan.
  dvהljas
***************
*** 31959,31960 ****
--- 31959,32024 ----
  korsning/ADGvY
+ 
+ # Additions by Stefan:
+ SEK
+ adr.
+ anm.
+ art.
+ aug.
+ bl.a.
+ d.v.s.
+ dec.
+ dr
+ dvs.
+ e.Kr.
+ e.d.
+ enl.
+ etc.
+ f.Kr.
+ f.d.
+ f.n.
+ f.צ.
+ febr.
+ fig.
+ fil.kand.
+ fil.lic.
+ forts.
+ fr.o.m.
+ fre.
+ jan.
+ jfr
+ kap.
+ kl.
+ lצr.
+ m.fl.
+ m.m.
+ max.
+ min.
+ mוn.
+ ngn
+ ngt
+ nov.
+ nr.
+ o.s.v.
+ obs.
+ okt.
+ ons.
+ osv.
+ p.g.a.
+ resp.
+ s.k.
+ sept.
+ sצn.
+ t.ex.
+ t.o.m.
+ tab.
+ tis.
+ tors.
+ u.a.
+ uppl.
+ utg.
+ v.g.
+ v.g.v.
+ vard.
+ vd
+ vol.
  korsningsfri/OY
