*** de_19.orig.aff	Mon Aug 15 22:45:35 2005
--- de_19.aff	Mon Aug 15 22:54:10 2005
***************
*** 3,4 ****
--- 3,24 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗ÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD	'
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
+ 
  
