*** de_AT.orig.aff	Thu Aug 25 11:22:16 2005
--- de_AT.aff	Thu Aug 25 11:22:16 2005
***************
*** 3,4 ****
--- 3,24 ----
  
+ FOL  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ LOW  אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿ
+ UPP  ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ‗ÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD	'
+ 
+ MAP 9
+ MAP aאבגדהו
+ MAP eטיךכ
+ MAP iלםמן
+ MAP oעףפץצ
+ MAP uשתûü
+ MAP nס
+ MAP cח
+ MAP yÿ‎
+ MAP s‗
+ 
  
*** de_AT.orig.dic	Thu Aug 25 11:22:16 2005
--- de_AT.dic	Thu Aug 25 11:24:01 2005
***************
*** 18,20 ****
  Fleischbהnke/N
- Fleischbank
  Fleischhauer/NS
--- 18,19 ----
***************
*** 151,153 ****
  zulieb
- 77857
  Äbte/N
--- 150,151 ----
***************
*** 18792,18794 ****
  Geschwulstherde
- Geselchte/N
  Geselle/N
--- 18790,18791 ----
***************
*** 20472,20474 ****
  HTML
- Hהfen
  Hהftling/EPS
--- 20469,20470 ----
