*** br_FR.aff.orig	2012-08-01 22:40:24.000000000 +0200
--- br_FR.aff	2012-08-01 22:46:03.028851510 +0200
***************
*** 13,20 ****
  SET UTF-8
  TRY esiaùnñrtolcdugmphbyfvkwzESIAÙNÑRTOLCDUGMPHBYFVKWZ'
  
  # Permet à l'utilitaire en ligne de commande hunspell de fonctionner avec les mots contenant des quotes (lec'hienn, ...)
! WORDCHARS '
  
  FLAG long
  
--- 13,22 ----
  SET UTF-8
  TRY esiaùnñrtolcdugmphbyfvkwzESIAÙNÑRTOLCDUGMPHBYFVKWZ'
  
+ MIDWORD '
+ 
  # Permet à l'utilitaire en ligne de commande hunspell de fonctionner avec les mots contenant des quotes (lec'hienn, ...)
! WORDCHARS '-
  
  FLAG long
  
