*** nl_NL.orig.aff	2010-08-06 21:51:58.000000000 +0200
--- nl_NL.aff	2010-08-06 22:04:57.000000000 +0200
***************
*** 1,9 ****
  SET ISO8859-1
  TRY esianrtolcdugmphbyfvkwjkqxz-כיטךןמהאגצüû'ESIANRTOLCDUGMPHBYFVKWJKQXZ
  
  # combined layout for BE and NL keyboards NL en BE
! KEY qwertyuiop|asdfghjkl|zxcvbnm|qawsedrftgyhujikolp|azsxdcfvgbhnjmk|aze|qsd|lm|wx|aqz|qws|
  
  # wordchars is (or is supposed to be) only for command line
! WORDCHARS '
  # . is not useful as wordchar
--- 1,23 ----
  SET ISO8859-1
+ 
+ NAME 	Nederlands
+ AUTHOR	Various, edited by Bram Moolenaar
+ VERSION	1.1.2
+ 
  TRY esianrtolcdugmphbyfvkwjkqxz-כיטךןמהאגצüû'ESIANRTOLCDUGMPHBYFVKWJKQXZ
  
+ FOL  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ LOW  ‗אבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏ÿ
+ UPP  ‗ÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞÿ
+ 
+ SOFOFROM abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZאבגדהוזחטיךכלםמןנסעףפץצרשתûü‎‏‗ÿÀÁÂÃÄÅÆÇÈÉÊËÌÍÎÏÐÑÒÓÔÕÖØÙÚÛÜÝÞ¿
+ SOFOTO   ebctefghejklnnepkrstevvkesebctefghejklnnepkrstevvkeseeeeeeeceeeeeeeedneeeeeeeeeeepseeeeeeeeceeeeeeeedneeeeeeeeeeep?
+ 
+ MIDWORD	'-
+ 
  # combined layout for BE and NL keyboards NL en BE
! #KEY qwertyuiop|asdfghjkl|zxcvbnm|qawsedrftgyhujikolp|azsxdcfvgbhnjmk|aze|qsd|lm|wx|aqz|qws|
  
  # wordchars is (or is supposed to be) only for command line
! #WORDCHARS '
  # . is not useful as wordchar
***************
*** 14,16 ****
  # do not offer wplit words (English desease)
! NOSPLITSUGS
  
--- 28,30 ----
  # do not offer wplit words (English desease)
! #NOSPLITSUGS
  
***************
*** 20,23 ****
  # added to make this dictonary work on planned version 3.2 of OpenOffice.org too.
! BREAK 1
! BREAK -
  
--- 34,37 ----
  # added to make this dictonary work on planned version 3.2 of OpenOffice.org too.
! #BREAK 1
! #BREAK -
  
***************
*** 186,193 ****
  # make calculation of alternatives better for words with accents
! MAP 5
! MAP eיטכך
! MAP aבא
! MAP iםלןמ
! MAP oףעצפ
! MAP uüתשû
  
--- 200,217 ----
  # make calculation of alternatives better for words with accents
! #MAP 5
! #MAP eיטכך
! #MAP aבא
! #MAP iםלןמ
! #MAP oףעצפ
! #MAP uüתשû
! MAP 9
! MAP aאבגדהו
! MAP eטיךכ
! MAP iלםמן
! MAP oעףפץצ
! MAP uשתûü
! MAP nס
! MAP cח
! MAP yÿ‎
! MAP s‗
  
***************
*** 508,509 ****
  REP spercibo sperziebo
! REP voorzover voor_zover
\ No newline at end of file
--- 532,534 ----
  REP spercibo sperziebo
! REP voorzover voor_zover
! REP hopenlijk hopelijk
